app/routes/__index/quest/sign/$user/$tokenId.tsx

```diff
@@ -0,0 +1,38 @@
+import { ec } from "starknet";
+import { pedersen } from "starknet/dist/utils/hash";
+import { sign } from "starknet/dist/utils/ellipticCurve";
+
+import eligibleUsers from "../../../../../../public/eligibleUsers.json";
+import type { LoaderArgs } from "@remix-run/node";
+
+
+export async function loader({ params }: LoaderArgs) {
+    const user = params.user
+    const privateKey = process.env.QUEST_SIGNER_PRIVATE_KEY;
+    const starkKeyPair = ec.getKeyPair(privateKey || '0x0');
+    const res = {
+        low: '',
+        high: ''
+    };
+    if (user) {
+        for (let index = 0; index < eligibleUsers.length; index++) {
+            const token = eligibleUsers[index];
+            const whitelistedUser = token.user;
+            const tokenId = token.token_id;
+            if (tokenId.toString() == params.tokenId) {
```

Comment:
As you casted your variable you should use === instead of == to ensure that the data have the same type.

Reply:
done @Marchand-Nicolas